CHANGELOG.md

```diff
@@ -15,9 +15,11 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
     _Security_ in case of vulnerabilities.
  -->
 
-## [Unreleased](https://github.com/o1-labs/o1js/compare/fbd4b2717...HEAD)
+## [0.12.3](https://github.com/o1-labs/o1js/compare/fbd4b2717...HEAD)
```

Comment:
could we make this follow the standard and include a specific commit instead of HEAD?

Reply:
Sure, thank you.